kernel/packages/shared/profiles/transformations/analizeColorPart.ts

```diff
@@ -1,15 +1,16 @@
+import { ReadOnlyColor4 } from 'decentraland-ecs/src'
 import { convertToRGBObject } from './convertToRGBObject'
-export function analizeColorPart(avatar: any, ...alternativeNames: string[]) {
+export function analizeColorPart(avatar: any, ...alternativeNames: string[]): Optional<ReadOnlyColor4, 'a'> {
   for (let name of alternativeNames) {
     if (!avatar[name]) {
       continue
     }
     if (typeof avatar[name] === 'string') {
       if (avatar[name].length === 7) {
-        return { color: convertToRGBObject(avatar[name]) }
+        return convertToRGBObject(avatar[name])
```

Comment:
In this cases we were returning something of the type:
```
{ color: { r, g, b} } 
```

Below, we were returning 
```
{ r, g, b}
```